Source/quests.cpp

```diff
@@ -833,14 +833,18 @@ void StartQuestlog()
 		}
 	}
 
-	selectedEntry = 0;
+	if (fistfinishedEntry != 0) {
+		selectedEntry = 0;
+	} else {
+		selectedEntry = qlistCnt;
+	}

```

Comment:
```suggestion
	selectedEntry = (fistfinishedEntry == 0 ? qlistCnt : 0);
```
also you have a typo, should be firstFinishedEntry, not fistfinishedEntry

Reply:
oh - you are right - thanks to auto completion, I never noticed ;)